atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractSerializableTransactionTest.java

```diff
@@ -532,6 +534,27 @@ public void testColumnSelection2() {
                 RangeRequest.builder().retainColumns(ImmutableList.of(PtBytes.toBytes(col))).build());
     }
 
+    @Test
+    public void testColumnRangeReadUnsupported() {
+        Transaction t1 = startTransaction();
+        try {
+            t1.getRowsColumnRange(TEST_TABLE, ImmutableList.of(PtBytes.toBytes("row1")),
+                    new ColumnRangeSelection(PtBytes.EMPTY_BYTE_ARRAY, PtBytes.EMPTY_BYTE_ARRAY), 1);
```

Comment:
There needs to be a `fail()` after this line, otherwise not throwing also passes